### PR TITLE
Revise ninechronicles-staked-and-dcc [ninechronicles-staked-and-dcc]

### PR DESCRIPTION
**Pull request by longfin**
This PR revises `ninechronicles-staked-and-dcc` strategy (introduced in #901), to use [Wrapped Nine Chronicles Gold(wNCG)][wNCG] instead of its LP token. (thanks @lounlee)

I'm still newer to snapshot strategies, so please let me know if there are any more fixes or things to consider when version bumping.

[wNCG]: https://etherscan.io/token/0xf203ca1769ca8e9e8fe1da9d147db68b6c919817

### CODE REVIEW

File: src/strategies/ninechronicles-staked-and-dcc/README.md [hunk 1]
```diff
@@ -1,33 +1,47 @@
...
+- `ethWNCGAddress` - (**Required**, `string`): The address of wNCG token contract on Ethereum.
+- `ethBalancerVaultAddress` - (**Required**, `string`): The address of Balancer Vault contract on Ethereum.
 - `ethDccAddress` - (**Required**, `string`): The address of D:CC token contract on Ethereum.
 - `ncGraphQLEndpoint` - (**Required**, `string`): The GraphQL endpoint of Nine Chronicles to query.
 - `ncBlockHash` - (**Required**, `string`): The target Block Hash of Nine Chronicles.
-- `lpTokenDecimal` - (**Optional**, `number`): The decimal precision for staked token. default is 18.
+- `wNCGDecimal` - (**Optional**, `number`): The decimal precision for wNCG. default is 18.
 - `weights` - (**Optional**, `number`): Weight values for the fomular . these values must be integers without decimal parts.
   - `stakedNCG`: Weight for staked NCG. (`w1`)
   - `stakedLPToken`: Weight for staked LP token. (`w2`)
```
Comment (lounlee) [inline src/strategies/ninechronicles-staked-and-dcc/README.md:47]: stakedLPToken -> stakedWNCG?

Comment (longfin) [reply src/strategies/ninechronicles-staked-and-dcc/README.md]: Addressed on https://github.com/snapshot-labs/snapshot-strategies/pull/914/commits/acd08cc9a4323182c72efe9139adf8d98dcb626d.

**Review by lounlee — APPROVED** [review]
LGTM 👍

**Review by ChaituVR — APPROVED** [review]
✅ No open proposals

**Pull request merged**